pkg/events/parse_args.go

```diff
@@ -198,17 +198,51 @@ func ParseArgs(event *trace.Event) error {
 				ParseOrEmptyString(modeArg, inodeModeArgument, err)
 			}
 		}
-	case SecuritySocketSetsockopt, Setsockopt, Getsockopt:
+	case SecuritySocketSetsockopt, Setsockopt:
 		if levelArg := GetArg(event, "level"); levelArg != nil {
 			if level, isInt := levelArg.Value.(int32); isInt {
 				levelArgument, err := helpers.ParseSocketLevel(uint64(level))
 				ParseOrEmptyString(levelArg, levelArgument, err)
 			}
 		}
+		customOptionsStringsMap := map[uint64]string{
```

Comment:
This will be reallocated for each relevant event, we had the same pattern in libbpfgo and it can cause a bottleneck. It was solved by declaring this map globally (see https://github.com/aquasecurity/libbpfgo/pull/130).